org.scala-ide.sdt.core.tests/src/org/scalaide/core/text/DocumentTest.scala

```diff
@@ -216,4 +216,46 @@ class DocumentTest {
   @Test
   def lastOpt_on_empty_file_returns_none() =
     document("").lastOpt === None
+
+  @Test
+  def lineInformation_on_non_empty_file_succeeds() = {
+    val d = document("""|first
+                        |second
+                        |third
+                        |""".stripMargin)
+    d.lineInformation(1).text(d) === "second"
+  }
+
```

Comment:
for me is ok of course, but I thought here about Kwestor's rule that 'all
public methods should have full signatures' :).

2015-05-29 15:27 GMT+02:00 Simon Schäfer <notifications@github.com>:

> In
> org.scala-ide.sdt.core.tests/src/org/scalaide/core/text/DocumentTest.scala
> <https://github.com/scala-ide/scala-ide/pull/886#discussion_r31323580>:
>
> > @@ -216,4 +216,46 @@ class DocumentTest {
> >    @Test
> >    def lastOpt_on_empty_file_returns_none() =
> >      document("").lastOpt === None
> > +
> > +  @Test
> > +  def lineInformation_on_non_empty_file_succeeds() = {
> > +    val d = document("""|first
> > +                        |second
> > +                        |third
> > +                        |""".stripMargin)
> > +    d.lineInformation(1).text(d) === "second"
> > +  }
> > +
>
> the return type is inferred if you mean that
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scala-ide/scala-ide/pull/886/files#r31323580>.
>